AltServer/AppDelegate.swift

```diff
@@ -37,16 +37,10 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         ConnectionManager.shared.start()
         
         let item = NSStatusBar.system.statusItem(withLength: -1)
-        guard let button = item.button else { return }
-        
-        button.image = NSImage(named: "MenuBarIcon")
-        button.target = self
-        button.action = #selector(AppDelegate.presentMenu)
-        
+        item.menu = self.appMenu
+        item.button!.image = NSImage(named: "MenuBarIcon")
```

Comment:
I have no idea when this would ever possibly be nil, but I can revert this if you want.

Reply:
Since I'm already using `guard-let`, I don't see a reason to change it to be force unwrap.

Reply:
Returning if item.button is nil doesn't seem right to me, since it silently skips the rest of the method which seems to be doing important things. What do you think of:
```suggestion
	if let button = item.button
	{
	        button.image = NSImage(named: "MenuBarIcon")
	}
```